lms/static/js/student_account/views/LoginView.js

```diff
@@ -113,6 +113,11 @@
                 this.setErrors();
                 this.element.hide( this.$resetSuccess );
 
+                // Labster set a new error message title on UK region server
+                if (msg.toString().indexOf('https://courses-uk.labster.com')!=-1){
+                    this.$errors.find('.message-title').text('We could not sign you in. Wrong Region?')
```

Comment:
Isn't it better to change text here? https://github.com/Livit/Livit.Learn.EdX/blob/labster/develop/lms/templates/student_account/login.underscore#L20

Reply:
I don't think so @TamoshaytisV ,
I make like you said in the previous commit but I realize when error login on UK region the message still same although the student on UK region, is that okay ?